cmd/create.go

```diff
@@ -2,14 +2,17 @@ package cmd
 
 import (
 	"bytes"
+	"crypto/ed25519"
 	"crypto/rand"
 	"crypto/rsa"
 	"crypto/x509"
 	"encoding/pem"
 	"os"
 	"runtime"
+	"strings"
 	"sync"
 
+	"github.com/mikesmitty/edkey"
```

Comment:
This mod hasn't been up modified in 6 years. I think it would be better to copy over the code so we can at least maintain it. Will refer to Jake though on what he thinks is best.

Reply:
Even though the code is MIT we should never copy code wholesale without attribution. Also, we should prefer forking if a repo isn't properly maintained, and only if we plan to modify it in some way.

Reply:
Sounds good Jake understood. Well in this instance the repo doesn't have a go.mod file, it's causing failures in the validate checks for the PR. Do you recommend forking and adding a go.mod for our purposes?

Reply:
Did a little digging, according to the docs `x509.MarshalPKCS8PrivateKey` should work: https://cs.opensource.google/go/go/+/go1.20.5:src/crypto/x509/pkcs8.go;l=101 

> MarshalPKCS8PrivateKey converts a private key to PKCS #8, ASN.1 DER form.
> 
> The following key types are currently supported: *rsa.PrivateKey, *ecdsa.PrivateKey, ed25519.PrivateKey (not a pointer), and *ecdh.PrivateKey. Unsupported key types result in an error.

This way we can use the go standard library and not have an external dependency at all.

Reply:
@slickwarren Circling back on this, were you able to test the above method?

Reply:
I tried that at the time of original testing, but it was not working as expected for me. I can try this again after the upcoming release to make sure I didn't miss anything. But I worked at it for awhile before finding this alternative.

Reply:
okay, it appears that this works if you cast the key to its expected type before sending to PKCS8, tested and working with both ed25519 and rsa keys now without the custom code